redash/monitor.py

```diff
@@ -36,4 +36,14 @@ def get_status():
         'data_sources': ''
     }
 
+    status['database_metrics'] = []
+    # have to include the fake FROM in the SQL to prevent an IndexError
+    queries = [
+        ['Query Results Size', "pg_size_pretty(pg_total_relation_size('query_results')) as size from (select 1) as a"],
+        ['Redash DB Size', "pg_size_pretty(pg_database_size('postgres')) as size from (select 1) as a"]
```

Comment:
How about we return the value of `pg_total_relation_size` and convert it to human readable format in the UI (using the `prettySize` Angular filter)? This way the values can be used in monitoring scripts.